### ISSUE TITLE
IE 11 toString - Invalid calling object

### ISSUE THREAD

**Issue by waynebrantley** [localForage/localForage-observable#22]
On IE11 when calling toString it fails in this code with 'Invalid calling object'.

```
function isDate(value) {
   return toString.call(value) === '[object Date]';
}
```
https://stackoverflow.com/questions/19321938/invalid-calling-object-error-in-ie

the fix is relatively easy it appears from this:
https://github.com/mobxjs/mobx/pull/1359/commits/f8ab36c13bd64d04d152a1776321540f1087ac71

simply define redefine the implicit window.toString here as ``var toString = Object.prototype.toString``

**Comment by waynebrantley** [comment]
@tofumatt or @thgreasi Can we look at getting this fixed?  It is a one line change.
I can do a PR or whatever, if you would be willing to review/merge and publish to npm?

**Comment by tofumatt** [comment]
Hi @waynebrantley, thanks for your patience in my reply! 😅

I'd be happy to merge that PR if you pushed a fix for it 👍

And we should be able to release a patch release afterward. Feel free to ping me again when it's ready 😄

**Comment by waynebrantley** [comment]
@tofumatt here is the PR.

Note - this project unchanged gave lots of typescript warnings and in general could probably use a refresh - FYI.  However, all tests did pass and still do.
Thanks!

**Comment by tofumatt** [comment]
Closed by #23 and published to npm 😄
